stripes-web/src/test/java/org/stripesframework/web/mock/MockHttpSession.java

```diff
@@ -47,12 +48,22 @@ public MockHttpSession( ServletContext context ) {
    @Override
    public Object getAttribute( String key ) { return _attributes.get(key); }
 
+   @Override
+   public Object getValue( String s ) {
+      return null;
+   }
```

Comment:
Previously we implemented that with `getAttribute`?

Reply:
I have made the implementations of value-centric methods delegate to their attribute-centric counterparts.